index.php

```diff
@@ -0,0 +1,161 @@
+		$stmt_2->execute([':PostID' => $row['id']]);
+		while ($row_1 = $stmt_2->fetch(PDO::FETCH_ASSOC)) {
+			$comments[] = $row_1;
+		}
+		$row['comments'] = $comments;
+		$likes = intval($stmt_1->fetchColumn());
+		$row['likes'] = $likes;
+        $posts[] = $row;
+    }
+
+	if (empty($_POST)) {
+    }
+	elseif (isset($_POST['logout'])) {
+		$userFeedback = "Wylogowałeś się!";
+		unset($_SESSION['id']);
+		unset($_SESSION['email']);
+	}
+	else {
+		if(isset($_POST['login']) && isset($_POST['password'])) {
+
+			$login = $_POST['login'];
+			$password = $_POST['password'];
+
+			$stmt = $dbh->prepare("SELECT * FROM a30_Users WHERE login = :login");
+			$stmt->execute([':login' => $login]);
+			$user = $stmt->fetch(PDO::FETCH_ASSOC);
+
+			if($user) {
+				if(password_verify($password, $user['password'])) {
+					$_SESSION['id'] = $user['id'];
+					$_SESSION['login'] = $user['login'];
+					$_SESSION['email'] = $user['email'];
+					$userFeedback = "";
+				}
+				else {
+					$userFeedback = "Nie poprawne hasło!";
+				}
+			}
+			else {
+				$userFeedback = "Nie ma takiego użytkownika!";
+			}
+		}
+	}
+
+	if (isset($_SESSION['id'])) {
+
+		$stmt = $dbh->prepare("SELECT COUNT(*) FROM a30_Post WHERE userID = :userID");
+		$stmt->execute([':userID' => $_SESSION['id']]);
+		$ile_photos = intval($stmt->fetchColumn());
+
+		$stmt = $dbh->prepare("SELECT * FROM a30_Post WHERE userid = :userid ORDER BY createdTime DESC LIMIT 1");
+    	$stmt->execute([':userid' => $_SESSION['id']]);
+    	$last_post = $stmt->fetch(PDO::FETCH_ASSOC);
+	}
+
+	if(isset($_GET['like']) && isset($_SESSION['id'])){
+
+			$id = intval($_GET['like']);
+
+			$stmt = $dbh->prepare("SELECT COUNT(*) FROM a30_User_Post_Like WHERE userID = :userID AND PostID = :PostID");
+			$stmt->execute([':userID' => $_SESSION['id'], ':PostID' => $id]);
+			$how_many_likes = intval($stmt->fetchColumn());	
+	
+			if ($how_many_likes == 0) {
+				try {
+					$stmt = $dbh->prepare ("INSERT INTO a30_User_Post_Like (
+					id, userID, PostID) 
+					VALUES (
+						null, :userID, :PostID) 
+					");
+	
+				$stmt->execute([':userID' => $_SESSION['id'], ':PostID' => $id]);
+				} 
+				catch (PDOException $e) {
+				}
+				$userFeedback = "Lubisz zdjęcie!";
+				header('Location: /main');
+				exit();
+			}
+			else {
+				header('Location: /main');
+				exit();
+			}
+	}
+	
+	if (isset($_GET['comments']) && isset($_SESSION['id']) && isset($_POST['comment']) ) {
+
+		$id = intval($_GET['comments']);
+		$comment = $_POST['comment'];
+		
+		if (mb_strlen($comment) >= 2 && mb_strlen($comment) <= 200) {
+			try {
+				$stmt = $dbh->prepare ("INSERT INTO a30_User_Post_Comment (
+				id, userID, postID, comment, createdTimeComment) 
+				VALUES (
+					null, :userID, :postID, :comment, '$currentDate') 
+				");
+	
+			$stmt->execute([':userID' => $_SESSION['id'], ':postID' => $id, ':comment'=> $comment]);
+			} 
+			catch (PDOException $e) {
```

Comment:
This is also applicable to other places with errors:

consider sending header with different response code than 200 (sometimes 4xx, sometimes 5xx (like here))